sections/main-cart-items.liquid

```diff
@@ -116,48 +116,43 @@
                     </p>
                   </td>
 
-                  <td class="cart-item__prices right">
+                  <td class="cart-item__totals right medium-down">
                     <div class="cart-item__price-wrapper">
-                      {%- if item.original_price != item.final_price -%}
+                      {%- if item.original_line_price != item.final_line_price -%}
                         <dl class="cart-item__discounted-prices">
                           <dt class="visually-hidden">
                             {{ 'products.product.price.regular_price' | t }}
                           </dt>
                           <dd>
                             <s class="cart-item__old-price price price--end">
-                              {{ item.original_price | money }}
+                              {{ item.original_line_price | money }}
                             </s>
                           </dd>
                           <dt class="visually-hidden">
                             {{ 'products.product.price.sale_price' | t }}
                           </dt>
                           <dd class="price price--end">
-                            {{ item.final_price | money }}
+                            {{ item.final_line_price | money }}
                           </dd>
                         </dl>
-
-                        {%- if item.variant.available and item.unit_price_measurement -%}
-                          <div class="unit-price caption">
-                            <span class="visually-hidden">{{ 'products.product.price.unit_price' | t }}</span>
-                            {{ item.variant.unit_price | money }}
-                            <span aria-hidden="true">/</span>
-                            <span class="visually-hidden">&nbsp;{{ 'accessibility.unit_price_separator' | t }}&nbsp;</span>
-                            {%- if item.variant.unit_price_measurement.reference_value != 1 -%}
-                              {{- item.variant.unit_price_measurement.reference_value -}}
-                            {%- endif -%}
-                            {{ item.variant.unit_price_measurement.reference_unit }}
-                          </div>
-                        {%- endif -%}
```

Comment:
I'm not sure we should be removing this. I'm not sure if it's due to legal requirements in some countries or not but maybe we should double check before removing unit prices. 

Our docs say: 
![](https://screenshot.click/21-08-gmtmb-pygey.png)

Reply:
Good point, I forgot to mention this to @melissaperreault .

You are right, this is a legal requirement so we will need to display. I added back.

[![alt](https://screenshot.click/19-27-rccgf-tandb.png)](https://screenshot.click/19-27-rccgf-tandb.png)

Reply:
Good catch! It was removed because we took out the price column where it was before I believe, but anyway, great it's back!